server/src/controllers/eventController.ts

```diff
@@ -143,6 +149,90 @@ eventController.post('/checkout', async (req: Request, res: Response) => {
   }
 });
 
+/**
+ * @swagger
+ * /2/events/image-upload:
+ *   post:
+ *     summary: Upload image to google cloud storage, make public, return link
+ *     tags:
+ *     - New Event API
+  *     requestBody:
+ *       description: Image File
+ *     responses:
+ *       200:
+ *         description: Image successfully uploaded.
+ *         content:
+ *           application/json:
+ *             schema: {url: string}
+ *       400:
+ *         description: bad request
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 error:
+ *                   type: string
+ *                   description: Error message from the server.
+ *       500:
+ *         description: Internal Server Error. An error occurred while processing the request.
+ */
+
+eventController.post('/image-upload', upload.single('file'), async (req: Request, res: Response) => {
+  const gcloudKey = process.env.GCLOUD_KEY;
+  const bucketName = process.env.GCLOUD_BUCKET;
+
+  if (gcloudKey === undefined || gcloudKey === '') {
+    console.log('Bucket key undefined!');
+    return res.status(500).send('Bucket key undefined!');
+  }
+  if (bucketName === undefined || bucketName === '') {
+    console.log('Bucket name undefined!');
```

Comment:
I don't think the console.logs are necessary if the same message is being sent back in the response.

Reply:
removed these logs and made the logs in your other comments console.error